crates/api_models/src/user/dashboard_metadata.rs

```diff
@@ -46,6 +47,20 @@ pub struct ProcessorConnected {
     pub processor_name: String,
 }
 
+#[derive(Debug, serde::Deserialize, serde::Serialize)]
+pub struct OnboardingSurvey {
+    pub designation: String,
+    pub about_business: String,
+    pub business_website: String,
+    pub hyperswitch_req: String,
+    pub major_markets: Vec<String>,
+    pub business_size: String,
+    pub required_features: Vec<String>,
+    pub required_processors: Vec<String>,
+    pub planned_live_date: String,
+    pub miscellaneous: String,
```

Comment:
Please check once whether these parameters need to be Option

Reply:
Adressed